menu cleanups

jelischer opened the pull request:
see bug https://github.com/u-root/u-root/issues/3426

change da30c33df7676c8591504a0e839dc6a4ea8f028b (superseded)
and
change 57afea1171d25092172c95ca6f15f92457f50b46  (superseded)

AjanZhong commented:
@jelischer Hi Julian, we found an issue in ARM integration test. Now we have fixed this ARM integration test issue, would you mind help rebase your PR, then Github CI will trigger integration test again. Thanks.

jelischer commented:
> @jelischer Hi Julian, we found an issue in ARM integration test. Now we have fixed this ARM integration test issue, would you mind help rebase your PR, then Github CI will trigger integration test again. Thanks.

actually I wasted a day trying to do that and doing a signed-off entry and failed
I need help from someone who know what they are doing to hold my had through it..
especially since the original tree that I added the patches to has been modified and seems to hate me.

AjanZhong commented:
> should just delete my clone and restart from scratch and add the same patch to it? It seem to be the only way that I maybe able to get the signed-off-by into the pull request,, by making a new one from scratch.

I received your mail, I will provide steps for rebasing and add sign-off online in mail. :)

rminnich reviewed (commented):
my suggestion: there are six commits here. I think it need only be one. So
git rebase -I origin/main
change 5 of the six pick to squash.
do that part.
then git commit --amend -s
There's probably smarter ways to do it but this should work. Thanks.

rminnich commented:
ah I see some are merge commit, sorry, but the basic thing I suggest will still work. Just fewer 'pick' to change to 's'

jelischer commented:
Hey @rminnich, can you reapply the approval that was removed when I added the signed-of to the commit message?

jelischer commented:
@AjanZhong cany ou please review this, Ron seems to be away or something

AjanZhong commented:
> @AjanZhong can you please review this, Ron seems to be away or something

Sure, let me try to figure out root case of integration test failure first.

jelischer commented:
copilot says it is a missing certificate or similar:
